modules/claims_api/app/serializers/claims_api/auto_established_claim_serializer.rb

```diff
@@ -4,10 +4,27 @@ module ClaimsApi
   class AutoEstablishedClaimSerializer < EVSSClaimDetailSerializer
     attributes :token, :status, :evss_id, :flashes
 
+    attribute :evss_errors, if: -> { object.status == 'errored' }
+
     type :claims_api_claim
 
     def id
       object&.id
     end
+
+    def evss_errors
+      if object.evss_response&.any?
+        format_evss_errors(claim.evss_response['messages'])
```

Comment:
this is the line that error below is happening on it looks like
https://github.com/department-of-veterans-affairs/vets-api/pull/6554/files#r611799757

Reply:
ah, ok, this makes sense, the other test that is working doesnt have any evss errors listed so it never gets into this part of the serailizer

Reply:
ahh yup, forgot to switch `claim` to `object` in the serializer